easybuild/easyconfigs/i/icc/icc-2019.1.144-GCC-8.2.0-2.31.1.eb

```diff
@@ -8,6 +8,7 @@ description = "Intel C and C++ compilers"
 
 toolchain = {'name': 'dummy', 'version': 'dummy'}
 
+source_urls = ['http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/14865/']
 sources = ['parallel_studio_xe_%(version_major)s_update%(version_minor)s_composer_edition.tgz']
```

Comment:
`s/edition/edition_for_cpp`

Reply:
Ouch... That, and I need to fix the checksum too. On it.